tests/unit/test_python_splitter.py

```diff
@@ -196,6 +196,12 @@ def test_random_splitter(test_specs, python_dataset):
     for split in splits:
         assert set(split.columns) == set(python_dataset.columns)
 
+    # check values sum to 1
```

Comment:
one question, you mentioned `# check values sum to 1` but then you are not checking this. Did you want to check that the first split is 70%, etc?

Reply:
without the changes Le made this raises an error, so as long as it gets through this I'm happy. the other checks confirm the ratio is as expected.